src/pages/workspace/reimburse/WorkspaceReimburseNoVBAView.js

```diff
@@ -31,7 +31,8 @@ const WorkspaceReimburseNoVBAView = ({translate, policyID}) => (
             menuItems={[
                 {
                     title: translate('workspace.reimburse.viewAllReceipts'),
-                    onPress: () => openOldDotLink(`expenses?policyIDList=${policyID}&billableReimbursable=reimbursable&submitterEmail=%2B%2B`),
```

Comment:
This is a link to the **expenses** page and it's correct (I just added it in this [PR](https://github.com/Expensify/App/pull/5801)). We shouldn't be removing it.

Reply:
Oh yes, good catch not sure why I included this one.

Reply:
I know why, I changed the wrong one. It was supposed to in WorkspaceReimburseVBA.